docs/production/upgrade-or-modify.md

```diff
@@ -1,6 +1,6 @@
 # Upgrade or modify Zulip
 
-This page explains how to upgrade, patch, or modify Zulip, including:
+This page explains how to upgrade Zulip, including:
```

Comment:
"how to upgrade a Zulip server" would be nicer phrasing here; should probably be its own commit.

Reply:
Sounds good to me.